string_assignment.js

```diff
@@ -0,0 +1,153 @@
+// # String Exercises
```

Comment:
I see some instances of hard coding which is a bad practice. Please try to avoid them in future assignments. Before moving on to next problem, try to test your code with different inputs. Revisit some of comments below and try to solve those problems again. Otherwise keep up the good work and keep practicing!